src/content/docs/de/guides/typescript.mdx

```diff
@@ -72,7 +72,7 @@ Um zu überprüfen, ob das Plugin funktioniert, erstelle eine `.ts`-Datei und im
 
 ### UI-Frameworks
 
-Wenn dein Projekt ein [UI-Framework](/de/core-concepts/framework-components/) verwendet, können je nach Framework zusätzliche Einstellungen erforderlich sein. Weitere Informationen findest du in der TypeScript-Dokumentation deines Frameworks. ([Vue](https://vuejs.org/guide/typescript/overview.html#using-vue-with-typescript), [React](https://reactjs.org/docs/static-type-checking.html), [Preact](https://preactjs.com/guide/v10/typescript), [Solid](https://www.solidjs.com/guides/typescript))
+Wenn dein Projekt ein [UI-Framework](/de/core-concepts/framework-components/) verwendet, können je nach Framework zusätzliche Einstellungen erforderlich sein. Weitere Informationen findest du in der TypeScript-Dokumentation deines Frameworks. ([Vue](https://vuejs.org/guide/typescript/overview.html#using-vue-with-typescript), [React](https://react-typescript-cheatsheet.netlify.app/docs/basic/setup), [Preact](https://preactjs.com/guide/v10/typescript), [Solid](https://www.solidjs.com/guides/typescript))
```

Comment:
This is the only link I'm not sure about, since this looks to be more of a community project than an official React resource. I'd like to get @sarah11918's thoughts on this before merging. All the other links seem to work well for me, thanks @TheOtterlord 🙌

Reply:
Nice catch @Yan-Thomas ! When I follow the original (old) link, THEY THEMSELVES link to the React cheatsheet, so I think this is OK.

So take it away, Yan!